week0001/Week1.java

```diff
@@ -0,0 +1,12 @@
+public class Week1 {
```

Comment:
Please rename the file like this: `fizz_buzz_thao.java` or `FizzBuzzThao.java`. The same for JS & Python scripts.